bigtable_rs/src/bigtable.rs

```diff
@@ -285,15 +313,15 @@ impl BigTableConnection {
                         }
                     })
                     .collect();
-
+        
                 let auth_manager = Some(Arc::new(authentication_manager));
                 Ok(Self {
                     client: create_client(endpoints, auth_manager, is_read_only),
                     table_prefix: Arc::new(table_prefix),
                     timeout: Arc::new(timeout),
                 })
             }
-        }
+        }  
```

Comment:
I think if you use some editors it is usually easy to setup some auto clean up those white spaces while saving.

Otherwise, perhaps you can try run `cargo fmt` to see if those can be auto corrected.

Reply:
Run `cargo fmt` and pushed